src/pyvesync/vesyncfan.py

```diff
@@ -58,6 +58,13 @@
             'features': [],
             'mist_modes': ['auto', 'sleep', 'manual'],
             'mist_levels': list(range(1, 10))
+    },
+    'Superior6000S': {
+            'module': 'VeSyncSuperior6000S',
+            'models': ['LEH-S601S-WUS'],
+            'features': [],
+            'mist_modes': ['auto', 'humidity', 'sleep', 'manual'],
```

Comment:
in the vesync api, `auto` mode for this model is named `autoPro`. I saw this as an implementation detail and hid it so that the pyvesync api is consistent for all humidifiers (e.g. `set_humidity_mode('auto')`). Is this a good idea?